webui/src/lib/components/auth/forms.jsx

```diff
@@ -18,37 +19,49 @@ export const AttachModal = ({
   const search = useRef(null);
   const [searchPrefix, setSearchPrefix] = useState("");
   const [selected, setSelected] = useState([]);
+  const [results, setResults] = useState([]);
+  const [after, setAfter] = useState("");
+
+  const { response, error, loading } = useAPI(() => {
+      return searchFn(searchPrefix, after);
+  }, [searchPrefix, after]);
 
   useEffect(() => {
-    if (!!search.current && search.current.value === "")
-      search.current.focus();
-  });
+      if (response) {
```

Comment:
Why this is done under `useEffect`?
Why `results` are saved on the state, instead of simply using `response.results`?